src/smt/theory_lra.cpp

```diff
@@ -1586,12 +1588,49 @@ class theory_lra::imp {
 
     bool assume_eqs() {        
         svector<lpvar> vars;
+        m_var_value_table.clear();
```

Comment:
There is a table called m_model_eqs  It is intended to map impq values to variables. It leverages get_ivalue for the hash-function and value equality.